### PR TITLE
Fix requirements, clean up small test issues

### PR DESCRIPTION
**Pull request by SamTov**

### CODE REVIEW

File: znnl/models/jax_model.py [hunk 1]
```diff
@@ -123,7 +122,7 @@ def __init__(
 
         # Prepare NTK calculation
         self.empirical_ntk = nt.batch(
-            nt.empirical_ntk_fn(f=self._ntk_apply_fn, trace_axes=trace_axes),
+            nt.empirical_ntk_fn(f=jax.jit(self._ntk_apply_fn), trace_axes=trace_axes),
```
Comment (KonstiNik) [inline znnl/models/jax_model.py:125]: I think this is from an older version of main. 
Can you remove it?

File: znnl/training_strategies/simple_training.py [hunk 2]
```diff
@@ -369,7 +369,7 @@ def train_model(
         state = self.model.model_state
 
         loading_bar = trange(
-            1, epochs + 1, ncols=100, unit="batch", disable=self.disable_loading_bar
+            0, epochs, ncols=100, unit="batch", disable=self.disable_loading_bar
```
Comment (KonstiNik) [inline znnl/training_strategies/simple_training.py:372]: Is this relevant?

**Review by KonstiNik — CHANGES_REQUESTED** [review]
Nice PR! 
Very nice to the requirements correct. 

I think there are two unwanted changes, you could take a look at.

File: znnl/models/jax_model.py (hunk 1 above)
Comment (SamTov) [reply znnl/models/jax_model.py]: Done in [c5e1170](https://github.com/zincware/ZnNL/pull/117/commits/c5e1170613ec80b29d8b1b0acd2b5801520a8104)

File: znnl/training_strategies/simple_training.py (hunk 2 above)
Comment (SamTov) [reply znnl/training_strategies/simple_training.py:372]: If you want a recorder to only take the model's state at initialization, then yes, it is. I think this is the more flexible form and is, therefore, better.

**Review by KonstiNik — APPROVED** [review]

**Pull request merged**